doc/adr/0015-current-API-authentication-strategy.md

```diff
@@ -0,0 +1,116 @@
+# 0015. A description of the current API authentication strategy
+
+## Status
+
+Accepted
+
+## Problem
+
+We have no documentation around how authentication works in Weave Gitops. This will aim to capture the current state of
+the system.
+
+The 'decision' section has been skipped as the decisions have been made and reverse-engineering the rationale for them is unlikely to work. The 'context' section covers implementation details and the 'consequences' aims to enumerate major risks with the system as well as how to work with it safely.
```

Comment:
I understand that what we aim with the ADR is to document the architecture, correct ? do you manage to know where in the repo is suggested to use this format for that purpose? 

might be @ozamosi  knows ?

Reply:
I don't think there's any documentation about which format to use.

I chose to use an ADR as I'm comfortable with the format, it's already in use and that, hopefully, much of ADR-15 will be superseded by future architectural changes so by using an ADR those changes can be expressed cleanly as part of the ADR process.